modules/social_features/social_landing_page/templates/group--featured.html.twig

```diff
@@ -95,7 +95,9 @@
     </span>
   {% endif %}
 
-  <a href="{{ url }}" class="card__link" title="{{ label }}">
-    {% trans %}Read more{% endtrans %}
-  </a>
+  {% if not hide_card_link %}
+    <a href="{{ url }}" class="card__link" title="{{ label }}">
+      {% trans %}Read more{% endtrans %}
+    </a>
```

Comment:
Is there a reason we don't just generate this link in a preprocess hook and render it as `{{ read_more }}` ? Then if a preprocess hook wants to change it or hide it that can be done without introducing a special logic variable.

Reply:
This is done to make it easier to add new classes, wrappers, etc. by front-end developers

Reply:
I think it was just done like this before, and within this story, we didn't refactor templates themselves.
Oleksandr just adds a condition, which allows hiding read more link, which is relevant for the Dashboard.

Reply:
Okay : ) Maybe we can check in the future if we can improve this a little bit. I think from a `link` class a themer should be able to add classes if needed. I have the feeling that we're adding quite a few one-off flags like this in the template and would prefer to keep that logic out of the templates themselves and higher up in the theme layer.

Lets leave it as is for now :)